### ISSUE TITLE
Fullscreen Mode for images

### ISSUE THREAD

**Issue by pwilimzig** [ikuseiGmbH/smart-village-app-app#206]
It would be nice if images would have a full screen mode.

**Comment by digorath** [comment]
As I already mentioned to @pwilimzig:
We have clickable images with links already, so we need to evaluate how to move forward with that, if we decide to have a fullscreen mode for images.

**Comment by SoleCincis** [comment]
I totally agree. On my opinion, full screen images are a "better" feature (also from an accessibility point of view), could we consider the option of having an icon inside the fullscreen image that navigates to a link ? 
@pwilimzig

**Comment by donni106** [comment]
something pressable above something pressable is bad practice on mobile @SoleCincis.
i think it is more common in the case of multiple actions, that you have one pressable area that opens a modal or a bottom sheet, that gives the user multiples options to do, if there are. this would mean, pressing an image shows something new with the option to choose between "show full image" and "navigate to something the image links to"?!

**Comment by SoleCincis** [comment]
Sounds very good to me Daniel! 
My proposal was based on this question: how to close a full screen image to return to the previous screen? it's usually an "x" icon that user can press to close the full screen, that's why I thought having another icon in case the image has a link wouldn't be a problem
